UnityProject/Assets/Scripts/Shuttles/ShipThruster.cs

```diff
@@ -1,20 +1,24 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using Light2D;
 using UnityEngine;
 
 public class ShipThruster : MonoBehaviour
 {
 
 	public MatrixMove shipMatrixMove; // ship matrix move
 	public ParticleSystem particleFX;
+	public LightSprite lightSprite;
```

Comment:
I know everything else is public in this class because this is one of the older classes where that was fine, but new fields should always be private unless we really need to set and get it somewhere else (and even then, I'd suggest using properties instead).

Reply:
ok